flash_patcher/parse/patch_visitor.py

```diff
@@ -171,10 +172,56 @@ def visitAddAssetBlock(
         if not (self.decomp_location / remote_folder).exists():
             Path.mkdir(self.decomp_location / remote_folder)
 
-        shutil.copyfile(self.asset_folder / local_name, self.decomp_location / remote_name)
+        shutil.copyfile(self.folder / local_name, self.decomp_location / remote_name)
 
         self.modified_scripts.add(self.decomp_location / remote_name)
 
+    def visitExecPatcherBlock(
+        self: PatchfileProcessor,
+        ctx: PatchfileParser.ExecPatcherBlockContext
+    ) -> None:
+        """When we encounter a patch file, we should open and process it"""
```

Comment:
Prefer docstrings in the imperative mood. e.g. "Open and process a patch file when it is encountered."

https://peps.python.org/pep-0257/